build.sbt

```diff
@@ -262,7 +262,7 @@ lazy val runProj = (project in file("run"))
   .configure(addSbtIO, addSbtUtilLogging, addSbtCompilerClasspath)
 
 lazy val scriptedSbtProj = (project in scriptedPath / "sbt")
-  .dependsOn(commandProj)
+  .dependsOn(commandProj, sbtProj % "compile->test")
```

Comment:
We should not use Test configuration since it's a mess to publish them to Maven Central.